processing/src/main/java/org/apache/carbondata/processing/util/CarbonDataProcessorUtil.java

```diff
@@ -699,4 +701,33 @@ public static boolean isRawDataRequired(CarbonDataLoadConfiguration configuratio
     return iterators;
   }
 
+  public static int[] calcDimensionLengths(int numberOfSortColumns, int[] complexCardinality) {
+    if (!(numberOfSortColumns > 0)) {
```

Comment:
1. Rewrite the condition `if (!(numberOfSortColumns > 0))` as `if (numberOfSortColumns == 0)`
2. The functionality of this method is not clear. Add a comment to explain the logic explanation and use of this method